src/script/entity/message/JoinedAfterMLSMigrationFinalisationMessage.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import ko from 'knockout';
+
+import {SystemMessageType} from 'src/script/message/SystemMessageType';
+import {t} from 'Util/LocalizerUtil';
+
+import {SystemMessage} from './SystemMessage';
+
+export class JoinedAfterMLSMigrationFinalisationMessage extends SystemMessage {
+  constructor() {
+    super();
+    this.system_message_type = SystemMessageType.JOINED_AFTER_MLS_MIGRATION_FINALISATION;
+    this.caption = ko.pureComputed(() => t('conversationJoinedAfterMLSMigrationFinalisation'));
```

Comment:
if this always ends up with t('conversationJoinedAfterMLSMigrationFinalisation')
why is it a pure computed? I don't see any reason to use that here